src/components/templates/settingsLayout.tsx

```diff
@@ -1,16 +1,19 @@
 import React, { useEffect, useState } from "react";
 
 import { useTranslation } from "react-i18next";
-import { Outlet } from "react-router-dom";
+import { Outlet, useLocation } from "react-router-dom";
 
 import { LogoCatLarge, PageTitle } from "@components/atoms";
 import { Sidebar, TitleTopbar } from "@components/organisms";
 import { SettingsMenu } from "@components/organisms/settings";
 
 export const SettingsLayout = () => {
+	const { t: tSettings } = useTranslation("settings", { keyPrefix: "topbar" });
 	const { t } = useTranslation("global", { keyPrefix: "pageTitles" });
 	const [pageTitle, setPageTitle] = useState<string>(t("base"));
-	const { t: tSettings } = useTranslation("settings", { keyPrefix: "topbar" });
+	const { pathname } = useLocation();
+
+	const topbarTitle = pathname.startsWith("/settings") ? tSettings("personalSettings") : tSettings("title");
```

Comment:
I thought we wanted to make it like in the example: `/personal-settings` and `/organization-settings`. It would make the identification much easier, don't you think?

Reply:
never mind, `/settings` will work as well